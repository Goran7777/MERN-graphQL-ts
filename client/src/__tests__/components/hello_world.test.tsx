// import * as React from 'react';
// import { shallow } from 'enzyme';
// import Hello from '../../components/hello/Hello';

// describe('HelloWorld', () => {
//   it('renders', () => {
//     const wrapper = shallow(<Hello />);
//     expect(wrapper).toMatchSnapshot();
//   });
// });
