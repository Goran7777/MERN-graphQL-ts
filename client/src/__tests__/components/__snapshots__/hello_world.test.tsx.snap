// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld renders 1`] = `
<div>
  <h1
    className="main"
  >
    Hello!!!!
  </h1>
</div>
`;
